hack/cluster-openshift.sh

```diff
@@ -120,6 +120,10 @@ while [[ $# -gt 0 ]]; do
       WAIT_FOR_ISTIO=false
       shift
       ;;
+    -nj|--no-jaeger)
```

Comment:
This hack change wasn't already merged ? Perhaps this PR needs some rebase ?

Reply:
Yeah, needs a rebase.